README.md

```diff
@@ -21,10 +34,20 @@
 🤩 | Awesome [new features](https://trello.com/b/RZRnWRbI/revenuecat-product-roadmap)  
 
 
-## Installation
+## Getting Started
+For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
 
-Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to use the SDK
+Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to install the SDK.
+
+Or view our Android sample app:
+- [MagicWeather](examples/MagicWeather)
 
 ## Requirements
+- Java 8+
+- Minimum target: Android 4.0+ (API level 14+)
+
+## SDK Reference
+Our full SDK reference [can be found here](https://sdk.revenuecat.com/android/5.4.1/index.html).
```

Comment:
slight concern that this link will get outdated with every new version. 

The site is set up to redirect to the latest version, so you should be able to replace this with 
```suggestion
Our full SDK reference [can be found here](https://sdk.revenuecat.com/android/index.html).
```
and it'll be functionally the same